code/game/machinery/limbgrower.dm

```diff
@@ -201,7 +201,7 @@
 
 	for(var/datum/reagent/R in reagents.reagent_list)
 		dat += "[R.name]: [R.volume]"
-		dat += "<A href='?src=[REF(src)];disposeI=[R.id]'>Purge</A><BR>"
```

Comment:
REF(R) or name ?

Reply:
`[R]` expands to the type in case of datums, there was a missing text2path higher up though